### ISSUE TITLE
XViewer shows white space for images > 16386 pixels

### ISSUE THREAD

**Issue by MenthaSuaveolens** [linuxmint/xviewer#140]
```
 * Xviewer version (xviewer --version) Xviewer 2.6.2

 * Distribution - (Mint 17.2, Arch, Fedora 25, etc...) Linux Mint 20 Cinnamon
```

**Issue**
In case one of the dimensions of an image is greater than 16386 pixels then the area which is located after pixel 16386 is displayed as a white space either right (if the image is wider than16386 pixels) either in the bottom (if the image is taller than 16386 pixels)

**Steps to reproduce**
Create an image (eg. with GIMP) with one of the dimensions >> 16386. Export image as jpg or png. Try to view the image with Xviewer

**Expected behaviour** 
All contents should be scaled properly
![XViewerWhiteSpace](https://user-images.githubusercontent.com/47174957/119687070-2f45ef80-be47-11eb-9659-3d3ee5354f4b.jpg)
![test7](https://user-images.githubusercontent.com/47174957/119687118-3c62de80-be47-11eb-9099-979ce717c235.jpg)
![test7h](https://user-images.githubusercontent.com/47174957/119687172-471d7380-be47-11eb-98aa-771b7429416a.jpg)

**Other information**

**Comment by programmer-ceds** [comment]
This looks like a duplicate of issue #118 which was fixed by PR #121. Can you retest with a version of xviewer that is 2.8.1 or later? Then close this issue if the problem doesn't occur.

**Comment by MenthaSuaveolens** [comment]
Tested with version 2.8.3 => Problem solved!
